clients/python/scripts/test_cloud.py

```diff
@@ -0,0 +1,54 @@
+import os
+from PIL import Image
+import moondream as md
+
+
+def test_cloud_client():
+    # Initialize client
+    api_key = os.getenv("MOONDREAM_API_KEY")
+    if not api_key:
+        raise ValueError("MOONDREAM_API_KEY environment variable not set")
+
+    client = md.VL(api_key)
+
+    # Load a test image
+    image_path = "/Users/caleb/Projects/moondream/moondream/data/raven_01.webp"
```

Comment:
Change to the path of one of the images in this repo (assets folder in the root of the repository).